molo/yourwords/models.py

```diff
@@ -0,0 +1,26 @@
+from wagtail.wagtailcore.models import Page
+
+
+class Competition(Page):
+    name = Page.CharField(max_length=128)
```

Comment:
Will take it out, does Django not need this field though? Or will it come up automatically?